src/main-view/content/row-settings/send-keys-row-settings.js

```diff
@@ -110,6 +127,25 @@ class SendKeysRowSettings extends Gtk.Grid {
     // Signals & Properties
     this.repeatSwitch.connect('state_set', (self, state) => this.repeatChanged(state));
 
+    this.shortcutButton.connect('clicked', () => {
```

Comment:
It might be a good idea to move the code of the shortcut button to its own widget. In this way, we can reuse it also to replace `oppositeKeysEntry`.

It'd be nice if it could grab the keyboard entry (see `grabPointer` and `ungrabPointer` in `add-app-view/index.js`) shortcuts used by the desktop environment are not triggered. For example, pressing super in GNOME interferes with the app.